templates/layout.mustache

```diff
@@ -16,6 +16,7 @@
 
   <script type="text/javascript">
     {{{script}}}
+    ALLOW_LOOPING = {{#allowLooping}}true{{/allowLooping}}{{^allowLooping}}false{{/allowLooping}};
```

Comment:
We could, except that its not a declaration (that's up in `script.js`), so doing that would end up with double-declaring that variable. Also, since it's going into the window scope (per the way `script.js` is written), the `var` keyword is superfluous anyway and I wouldn't have bothered anywhere, except for matching style.